client/src/Footer.js

```diff
@@ -36,6 +36,23 @@ export const Footer = () => (
               >
                 Github.
               </a>
+              {""}
+              <br />
+              Thanks to {""}
+              <a
+                style={{ textDecoration: "underline", color: "white" }}
+                href="https://www.flaticon.com/authors/smashicons>Smashicons"
```

Comment:
Add a `title` attribute in the `<a>` tag with the name of the website

Reply:
i added it